src/CMakeLists.txt

```diff
@@ -41,6 +41,10 @@ if (MGCONSOLE_ON_WINDOWS)
   set(GFLAGS_WIN_LIB_SUFFIX "_static")
   set(CMAKE_CXX_STANDARD_LIBRARIES "-static-libgcc -static-libstdc++ -lws2_32")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-Bstatic,--whole-archive -lwinpthread -Wl,--no-whole-archive")
+elseif(MGCONSOLE_ON_OSX)
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static-libstdc++")
+elseif(UNIX AND NOT APPLE)
```

Comment:
I forgot about MGCONSOLE_ON_LINUX, you could use that. :sweat_smile:

Reply:
It's not a most precise name. What if start building for BSD? :D